### DISCUSSION TITLE
Updating French Translation

### DISCUSSION THREAD

**Discussion by sharky98** [kimai/kimai#4590] [General] [answered]
Hello,
I wanted to update the French translation, but I might have made some mistake in the usage of Weblate 🤦.

To whoever is going to correct my mistake, I am sorry 🙇!

I have put two source string as read-only (https://hosted.weblate.org/translate/kimai/messages/fr/?q=state:read-only) by trying to ask for more context of the source string.

If possible, it would be nice to have a very quick guide to make sure somebody else doesn't play around in the translation and break stuff in the [docs](https://www.kimai.org/documentation/translations.html) or maybe adding some controls if Weblate allows it (like a new user can only suggest translation, somebody with X translation approved can modify without approval, etc.)?

**Comment by kevinpapst** [comment, marked as answer]
I'll fix it. 
Even though I am not good with Weblate myself, but I manage my way around 😁

**Reply by sharky98** [reply]
Hello, I saw your comments on weblate. I prefer to answer here since I don't want to create more confusion there! First, when I wanted to ask question, I think I misused the 3 scopes values. Even now that I know what they are doing, I have hard time to make sense of the words related to the action they do 🤷‍♂️. Like, I understood (and still understand the wording) that the last one `Translation comment` is meant to be shared only with translators of the language you are working with and not everyone (hence my french comments). Similar thing for `Source string comment`, I though it would be the best selection to ask for clarification (while obviously it isn't the case!)

![image](https://github.com/kimai/kimai/assets/8150191/1d1474ae-078e-452f-9651-f93c95fae9dc)

So, that being said, regarding the [`timesheet.mode_duration_fixed_begin`](https://hosted.weblate.org/translate/kimai/system-configuration/en/?checksum=3d7303031362439b) translation; I never intended to change the English translation (probably used the wrong item in the above dropdown). I really apologize if you saw my action as such. As for the french translation; I didn't change it but put it in "suggestion" mode until hearing back from you (so to not lose the idea).

As for why I (tried) to ask for clarification, is because when using the UI only (not going into the code), the `configurable` part of the `fixed start-time` didn't make sense to me. Using that option, the dialog to start a time entry does not show the `start-time`; hence my confusion as how it could be configurable if it isn't shown to the user. But just know, while writing this up; I saw the options just below the drop-down (below both in the vertical and z-index axes).

![image](https://github.com/kimai/kimai/assets/8150191/50f7e7b6-6bad-47c6-a775-ca25eb68ee7f)

![image](https://github.com/kimai/kimai/assets/8150191/5a74f1d0-9199-4ac6-8453-1a19103c4473)

![image](https://github.com/kimai/kimai/assets/8150191/c5a9d013-0648-40c0-aa05-678274896741)

So, again, I apologize for the mess I made in Weblate. I think I might stay away from it 😅!

As for the `timesheet.mode_duration_fixed_begin`, if anything, I would suggest to review how the options are shown to avoid the potential confusion I experienced. Things like:
- having the `Default start-time` disabled if the `Timetracking mode` selected do not use it;
- having the field `Default start-time` indented to show the dependency of both options;
- rename the label of `Default start-time` to match the wording in the other option and be explicit of which timetracking mode make use of it to something like `Fixed start-time (for Duration timetracking mode)`).

But, if I remember when I looked at the code few months back; I think it would not be (easily) possible to do so. So, feel free to act as you wish on this one ☺️.

PS: Yeah, and if it wasn't obvious, I didn't [RTFM](https://www.kimai.org/documentation/configurations.html) 😂.

**Reply by kevinpapst** [reply]
As I said: I am not a weblate expert myself and don't know what these Weblate comment types are for.

See. I get around 20-50 emails every day related to Kimai. If I then suddenly receive 20 more emails from Weblate, it just makes my life harder. If my answer sound rude, I apologize. After answering dozens of questions each day, I might sound like a stressed out idiot, which I am likely be in that moment. Please don't take that personal, just try to see the other side of the coin.

I appreciate any input and help from the community. I do 💯 percent. All these translations wouldn't be possible without you and everyone else - Thank you 👍 

Please do not mix different topics in one issue. I will not answer more then the first topic.
